bundles/org.openhab.binding.salus/pom.xml

```diff
@@ -61,5 +59,11 @@
       <version>5.11.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
I thought we try to get rid of this dependency? It is in test scope, but why need it there?